### ISSUE TITLE
deleting other users' draft causes error

### ISSUE THREAD

**Issue by dorey** [kobotoolbox/kpi#447]
## Description

User can edit draft submissions without proper permissions.

## Steps to Reproduce

1. Create a draft submission as "User A"
2. Give "view submissions" permission to "User B"
3. Log in as "User B" and in Enketo load draft submission
4. Make changes and save --> observe changes are saved without any errors

## Expected behavior 

User shouldn't be able to change draft submissions if they have only "view submissions" permission.

## Old description:

if i have view permissions but not edit permission , the interface does not prevent me from deleting and the error is uncaught

**Comment by magicznyleszek** [comment]
@jnm this seems to be Enketo error? Or maybe we don't pass some permissions properly?

**Comment by jnm** [comment]
Right, this is the way Enketo works: drafts are stored only locally, within the browser, without any communication with the server.

The original bug that @dorey reported I believe had to do with deleting assets, but we tested this and found that it'd been fixed already. We also tested submission deletion for good measure, and the UI correctly does not display a delete button unless the user has appropriate permissions.
